index.html

```diff
@@ -173,6 +173,32 @@ <h2>Calls for implementation and feedback</h2>
 
 <p>When an addition is accepted at the “candidate” (stage 3) maturity level, the committee is signifying that it believes design work is complete and further refinement will require implementation experience, significant usage and external feedback.
 
+<h2>Blocking Advancement</h2>
+
+<p>During the discussion of a proposal, at any stage, any aspect may be discussed, including those outside of the scope of its current stage. A proposal’s advancement may be blocked, however the committee does not have a concept of a rejected proposal.</p>
+
+<p>To block the advancement of a proposal, a reason must be given and it must be discussed in committee. If no reason is given, the block is void for vagueness. If a delegate needs more time to consider aspects of a proposal, they may request a delay on the decision, or allow conditional advancement.</p>
+
+<p>Conditional advancement is time limited, giving the person raising the block time to discuss with the champions and authors about their concerns. If a proposal has conditional advancement, and issue is opened on the proposal’s repository. If the issue is resolved, the proposal automatically reaches the next stage without further discussion by the committee. If the issue cannot be resolved, the proposal does not advance.</p>
+
+<p>Blockers need to be well described and recorded within the committee, so that in future meetings and future proposals we can learn from failed proposals, and have a good record of why they failed.</p>
+
+<p>Regardless of stage, the following types of blockers require the committee to reach consensus in order for the block to be valid: Blockers without a given reason, Blockers which have been discussed at length by the committee and were resolved.</p>
+
+<p>Blocking criteria for reaching stage 1 are limited. Proposals which have fulfilled the acceptance criteria may not be blocked from reaching stage 1 unless there has been a very similar proposal that has already been through the process and has not advanced. In this case, the author(s) and champion(s) should be directed to the appropriate record of the previous proposal, so that they can revise their approach. The reason for this restricted ability to block is so that proposals have the necessary time to develop, and the committee has the necessary time to consider them.</p>
```

Comment:
@waldemarhorwat you had a comment here, that blocking criteria for stage 1 should **not** be limited. @bakkot had a similar, wider comment that blocking should not be too strictly defined.

My rationale here is, stage 1 acceptance signifies that "this is not a waste of the champion's/authors time" -- but also that it isn't at a point that the committee considers it a problem worth solving. Some issues may seem not very important at the moment, but could become more important as they are explored further. I think the problem with what I wrote here initially, is that it limits blocking to proposals that have been presented before. This is wrong, how about instead

```suggestion
<p>Blocking criteria for reaching stage 1 are limited. Proposals which have fulfilled the acceptance criteria may be blocked from reaching stage 1 if there are significant technical or foundational issues, or there has been a very similar proposal that has already been through the process and has not advanced for reasons that are not addressed in the new proposal. In the former case,  the author(s) and champion(s) should have as an outcome the reason for the block. In the latter case, the author(s) and champion(s) should be directed to the appropriate record of the previous proposal, so that they can revise their approach. The reason for this restricted ability to block is so that proposals have the necessary time to develop, and the committee has the necessary time to consider them.</p>
```

This is why I was thinking that stage 1 blocking should be limited. I am open to discuss others ideas about this, this has been my model of it.

Reply:
What is the problem we're trying to solve by making stage 1 blocking criteria more formal?

Reply:
I, and a number of other folks I've chatted with, don't agree with either the old or the new changes. Both are taking as a given that we should severely limit blocking criteria for stage 1. I don't see consensus that we should.

Reply:
Ok, based on this feedback, I will remove the text describing stage 1, and merge it with the "unlimited blocking" of stage 2 and stage 3. 

The goal of writing this criteria was to communicate better the convention that we have at committee. We very rarely block at stage 1. I am not sure how to get it across, it seems to vary by proposal, but that could be done in a second iteration, if this becomes and issue. I am fine with stage 1 being unlimited.